litestar/handlers/base.py

```diff
@@ -148,9 +148,7 @@ def __init__(
         self.type_encoders = type_encoders
 
         self.paths = (
-            {normalize_path(p) for p in path}
-            if path and isinstance(path, list)
-            else {normalize_path(path or "/")}  # type: ignore
+            {normalize_path(p) for p in path} if path and isinstance(path, list) else {normalize_path(path or "/")}  # type: ignore
```

Comment:
I make this 131 chars - maybe it won't break lines if only for a comment?

Reply:
same as above, i think you are right. I fought it but ended up confused and commited+pushed